tests/TestCase/src/Controller/PagesControllerTest.php

```diff
@@ -110,7 +110,7 @@ public function testAllSuperadminUrls()
 
         $this->assertPagesForErrors($testUrls);
 
-        $this->httpClient->doFoodCoopShopLogout();
+        $this->get($this->Slug->getLogout());
```

Comment:
Please create a new Method "logout" in new LoginTrait and simply call $this->logout()